homeassistant/components/cookidoo/manifest.json

```diff
@@ -0,0 +1,11 @@
+{
+  "domain": "cookidoo",
+  "name": "Cookidoo",
+  "codeowners": ["@miaucl"],
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/cookidoo",
+  "integration_type": "service",
+  "iot_class": "cloud_polling",
+  "quality_scale": "bronze",
+  "requirements": ["cookidoo-api==0.9.0"]
```

Comment:
Also, can we use `aiofiles` to access the localization files?

Reply:
done